staging/src/kubevirt.io/containerized-data-importer-api/pkg/apis/core/v1beta1/types.go

```diff
@@ -648,6 +648,12 @@ type VolumeImportSourceSpec struct {
 	Preallocation *bool `json:"preallocation,omitempty"`
 	// ContentType represents the type of the imported data (Kubevirt or archive)
 	ContentType DataVolumeContentType `json:"contentType,omitempty"`
+	// TargetClaim the name of the specific claim to be populated with a multistage import.
+	TargetClaim string `json:"targetClaim,omitempty"`
+	// Checkpoints is a list of DataVolumeCheckpoints, representing stages in a multistage import.
+	Checkpoints []DataVolumeCheckpoint `json:"checkpoints,omitempty"`
```

Comment:
I guess it might not be critical since its internal but maybe need to change the name of DataVolumeCheckpoint?, it is used from the import populators which seems weird

Reply:
yeah I think it's fine if it's just the types in the source.  Serialized structures shouldn't contain and datavolume stink